### DISCUSSION TITLE
问下作者现在是不是没有go语言版本的服务端做升级部署了?

### DISCUSSION THREAD

**Discussion by masktao** [SeaHOH/GotoX#136] [一般]
之前一直在用zebra, 9月底突然0 ip了,后来发现quic, tls1.3都无法开启,只有在tls1.2下可以用,但速度很慢.
去appengine页面提示:应用的某个版本正在使用已弃用的 Go 版本。请在 2019 年 10 月 1 日之前升级
是不是google的服务端已经升级,不接受老版本的客户端连接,需要升级到最新go语言的服务端..现在没有人开发新版本go语言的服务端和客户端.所以goproxy和zebra基本都死了吧?
现在只有python的gotoX和XX-net可以用,对吗?
需要重新上传python的服务端升级对么?

**Comment by SeaHOH** [comment]
我中文不行，看不懂你在讲什么。

**Comment by masktao** [comment]
> 我中文不行，看不懂你在讲什么。

抱歉说复杂了,之前一直用goproxy/zebra,现在不好用了
改用gotoX/xx-net是不是要重新部署服务端?

**Comment by SeaHOH** [comment]
自己看，都中文写的。
